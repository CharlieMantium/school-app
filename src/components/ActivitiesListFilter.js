import React from 'react';
import Input from './Input';

const ActivitiesListFilter = () => (
  <div>
    <Input type="text" />
  </div>
);

export default ActivitiesListFilter;
