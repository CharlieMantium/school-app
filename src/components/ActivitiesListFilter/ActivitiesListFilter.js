import React from 'react';

import Input from '../Input/Input';

const ActivitiesListFilter = () => (
  <div>
    <Input data-test="input-component" type="text" onChange={() => {}} />
  </div>
);

export default ActivitiesListFilter;
