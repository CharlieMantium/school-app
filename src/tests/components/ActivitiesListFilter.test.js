import React from 'react';
import { shallow } from 'enzyme';

import ActivitiesListFilter from '../../components/ActivitiesListFilter';

test('should render ActivitiesListFilter correctly', () => {
  const wrapper = shallow(<ActivitiesListFilter />);
  expect(wrapper).toMatchSnapshot();
});
