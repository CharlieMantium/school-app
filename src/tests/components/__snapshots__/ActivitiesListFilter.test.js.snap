// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ActivitiesListFilter correctly 1`] = `
<div>
  <Input
    errorMsg=""
    onBlur={null}
    onChange={[Function]}
    placeholder=""
    type="text"
    value=""
  />
</div>
`;
